python/lsst/dax/imgserv/locateImage.py

```diff
@@ -201,18 +255,21 @@ def __init__(self, credFileName, logger=log):
                        database="DC_W13_Stripe82",
                        table="Science_Ccd_Exposure",
                        columns=["run", "camcol", "field", "filterName"],
-                       dataRoot="/lsst7/stripe82/dr7/runs",
+                       dataRoot="/datasets/sdss/preprocessed/dr7/runs",
```

Comment:
I think all of these should probably be relocated to configuration files and the number of W13xxxDb classes can be reduced, but that's for a future refactor.